backend/services/logs/src/bullmq/afterProcess.ts

```
@@ -0,0 +1,261 @@
+type ProcessHandlerProps = {
+  subdomain: string;
+  pluginName: string;
+  rules: IAfterProcessRule[];
+  payload: any;
+  contentType?: string;
+  action: string;
+};
+
+function getAllKeys(obj, prefix = '') {
+  let keys: string[] = [];
+  for (let key in obj) {
```

ESLint

[error] 'key' is never reassigned. Use 'const' instead.

(prefer-const)